typescript_templates/model.vm

```diff
@@ -150,6 +202,36 @@ export class $def.name extends BaseModel {
 #end
     }
   }
+
+  static fromParsedJSON(data: Record<string, any>): $def.name {
```

Comment:
`from_obj_for_encoding` might be a more consistent name for this method, but it goes against our current naming conventions